homeassistant/components/switch/command_switch.py

```diff
@@ -22,22 +24,36 @@ def setup_platform(hass, config, add_devices_callback, discovery_info=None):
     devices = []
 
     for dev_name, properties in switches.items():
+        if 'statecmd' in properties and CONF_VALUE_TEMPLATE not in properties:
```

Comment:
We should support a state cmd based on return code of the command if no template specified